cmd/cmd-run.go

```diff
@@ -175,7 +176,22 @@ func run(cmd *cobra.Command, _ []string) error {
 		<-c
 		os.Exit(1)
 	}()
-
+	repoInclude, _ := flag.GetString("repo-include")
+	repoExclude, _ := flag.GetString("repo-exclude")
+	var repoIncludeFilterCompile *regexp.Regexp
+	var repoExcludeFilterCompile *regexp.Regexp
+	if repoInclude != "" {
+		repoIncludeFilterCompile, err = regexp.Compile(repoInclude)
```

Comment:
```suggestion
		repoIncludeFilterCompile, err := regexp.Compile(repoInclude)
```

Reply:
nit: Even if this doesn't have a difference here. Keeping the error in local scope is generally good practice.